extensions/blocks/google-calendar/edit.js

```diff
@@ -0,0 +1,196 @@
+			// the block becomes selected makes the overlap disappear too early. Hiding the overlay
+			// happens on mouseup when the overlay is clicked.
+			return { interactive: false };
+		}
+
+		return null;
+	}
+
+	hideOverlay = () => {
+		// This is called onMouseUp on the overlay. We can't respond to the `isSelected` prop
+		// changing, because that happens on mouse down, and the overlay immediately disappears,
+		// and the mouse event can end up in the preview content. We can't use onClick on
+		// the overlay to hide it either, because then the editor misses the mouseup event, and
+		// thinks we're multi-selecting blocks.
+		this.setState( { interactive: true } );
+	};
+
+	handleEmbed = event => {
+		if ( event ) {
+			event.preventDefault();
+		}
+		const { editedEmbed } = this.state;
+		const embedString = editedEmbed.trim();
+		let attributes;
+
+		if ( IFRAME_REGEX.test( embedString ) ) {
+			attributes = extractAttributesFromIframe( embedString );
+		} else {
+			attributes = { url: embedString };
+		}
+
+		if ( ! URL_REGEX.test( attributes.url ) ) {
+			this.setErrorNotice();
+			return;
+		}
+
+		this.props.setAttributes( attributes );
+		this.setState( { editingUrl: false, notice: null } );
+	};
+
+	setErrorNotice = () => {
+		this.setState( {
+			notice: __(
+				"Your calendar couldn't be embedded. Please double check your URL or code.",
+				'jetpack'
+			),
+		} );
+	};
+
+	getEditForm = ( className, editedEmbed ) => {
+		return (
+			<form onSubmit={ this.handleEmbed } className={ className }>
+				<textarea
+					type="text"
+					value={ editedEmbed }
+					className="components-placeholder__input"
+					aria-label={ __( 'Google Calendar URL or iframe', 'jetpack' ) }
+					placeholder={ __( 'Enter URL or iframe to embed here…', 'jetpack' ) }
+					onChange={ event => this.setState( { editedEmbed: event.target.value } ) }
+				/>
+				<Button isSecondary isLarge type="submit">
+					{ _x( 'Embed', 'button label', 'jetpack' ) }
+				</Button>
+			</form>
+		);
+	};
+	/**
+	 * Render a preview of the Google Calendar embed.
+	 *
+	 * @returns {object} The UI displayed when user edits this block.
+	 */
+	render() {
+		const { attributes, className } = this.props;
+		const { url } = attributes;
+		const { editedEmbed, interactive, editingUrl } = this.state;
+
+		const height = this.props.isMobile ? '300' : '500';
+
+		const html = `<iframe src="${ url }" style="border:0" scrolling="no" frameborder="0" width="100%" height=${ height }></iframe>`;
+
+		const permissionsLink = (
+			<ExternalLink href="https://en.support.wordpress.com/google-calendar/">
+				{ __( 'Enable Permissions for the calendar you want to share', 'jetpack' ) }
+			</ExternalLink>
+		);
+
+		const controls = (
+			<>
+				<InspectorControls>
+					<PanelBody title={ __( 'Calendar Settings', 'jetpack' ) } initialOpen={ false }>
+						{ this.getEditForm( `${ className }-embed-form-sidebar`, editedEmbed ) }
```

Comment:
This is causing issues when you use the custom css class field in the editor:

![image](https://user-images.githubusercontent.com/426388/74766077-fbb00900-5284-11ea-9ac2-58adbcb7d374.png)
![image](https://user-images.githubusercontent.com/426388/74766094-036fad80-5285-11ea-916c-c27931cd2bfd.png)

This is a problem a few lines down as well.

Related: #14623

Reply:
Added a fix for this, and let @pablinos know re #14623, will see if he thinks there is a better way to fix this or not.